manifests/2.5.0/opensearch-dashboards-2.5.0.yml

```diff
@@ -9,7 +9,7 @@ ci:
 components:
   - name: OpenSearch-Dashboards
     repository: https://github.com/opensearch-project/OpenSearch-Dashboards.git
-    ref: 2.x
+    ref: 2.5
```

Comment:
Hi @manasvinibs Could you use single quote for the ref here? python checks fail because it needs to be `'2.5'`.

Reply:
> Hi @manasvinibs Could you use single quote for the ref here? python checks fail because it needs to be `'2.5'`.

oh my bad! Updated!